aws/aws.go

```diff
@@ -796,7 +808,18 @@ func (s *awsOps) Create(
 		return nil, cloudops.NewStorageError(cloudops.ErrVolInval,
```

Comment:
It is not mandatory but being dereference below
`if *vol.VolumeType == opsworks.VolumeTypeIo1 || *vol.VolumeType == "gp3" {
`